academy/3-my-own-chain/game-winner.md

```diff
@@ -277,14 +277,6 @@ $ checkersd tx checkers play-move 1 1 2 2 3 --from $bob
 $ checkersd query checkers show-stored-game 1
 ```
 
-This should show:
-
-```
-...
-  winner: "*"
-...
-```
-
```

Comment:
This is not duplicate content. This is to show that this part of the content has not changed after a move.

Reply:
Please include it again. I was really unsure about that one.

Reply:
Included in https://github.com/cosmos/sdk-tutorials/pull/1169/files#diff-cf9cbdadfc7b04d5c082ee3dd04afc4c491f70826d5f8526750293020d33c748R384-R390 again.